hs_rhessys_modelinstance/models.py

```diff
@@ -0,0 +1,134 @@
+from lxml import etree
```

Comment:
This is a work in progress.  There are not yet migrations for it, and this app is not registered in hydroshare.settings, so it should be harmless to merge this now.